tekton_configs/pytest-task.yaml

```diff
@@ -0,0 +1,110 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: pytest
```

Comment:
pytest is too generic; this test is testing a notebook so maybe something like "test-notebook"

Reply:
yea, adding test-notebook task is definitely fine.  but I think having a pytest task would be great because we have many tests under pytests and we plan to migrate other tests to pytest as well, right?